### PR TITLE
[RELENG-151] allow for android-components githubscript workers

### PR DESCRIPTION
**Pull request by escapewindow**
We may want to wait for https://firefox-ci-tc.services.mozilla.com/tasks/fKCbYWfKSPa-pBmqpNh8Ow or similar task to go green. (That assumes `mobile-1-github-dev` spins up.)

### CODE REVIEW

File: githubscript/docker.d/worker.yml [hunk 1]
```diff
@@ -9,10 +9,15 @@ github_projects:
...
+        github_owner: mozilla-mobile
+        github_repo_name: android-components
     'COT_PRODUCT == "mobile" && ENV == "fake-prod"':
       mock:
         allowed_actions: ["release"]
         github_token: dummy
         github_owner: non-existing-owner
         github_repo_name: non-existing-repo
-taskcluster_scope_prefixes: ["project:mobile:fenix:releng:github:"]
+taskcluster_scope_prefixes: ["project:mobile:fenix:releng:github:", "project:mobile:android-components:releng:github:]
```
Comment (JohanLorenzo) [inline githubscript/docker.d/worker.yml]: Nit:

```suggestion
taskcluster_scope_prefixes: ["project:mobile:fenix:releng:github:", "project:mobile:android-components:releng:github:"]
```

**Review by JohanLorenzo — APPROVED** [review]
LGTM modulo nit! 👍

File: githubscript/docker.d/worker.yml (hunk 1 above)
Comment (escapewindow) [reply githubscript/docker.d/worker.yml]: That might be the reason mobile-1-github-dev didn't start up =\

**Comment by escapewindow** [comment]
Thanks!

**Pull request merged**